tensorflow/lite/acceleration/configuration/configuration.proto

```diff
@@ -627,6 +629,14 @@ message CPUSettings {
   optional int32 num_threads = 1 [default = -1];
 }
 
+// Arm NN Delegate Settings
+// More information about Arm NN delegate options can be found in https://arm-software.github.io/armnn/latest/delegate.xhtml
+message ArmNNSettings {
+  optional string backends = 1;
+  optional bool fastmath = 2;
+  optional string additional_parameters = 3;
```

Comment:
Please document this field.
E.g.
// Comma-separated list of name=value pairs, specifying additional ARM NN runtime options,
// e.g. "number-of-threads=4,gpu-tuning-level=3,enable-fast-math=true,logging-severity=Debug".
// See <https://arm-software.github.io/armnn/latest/runtimeoptions.xhtml>.
// Note that other fields in ArmNNSettings will take precedence over any corresponding parameters
// specified here, if both are set.